### PR TITLE
Changed name to Open Angel Arena

### PR DESCRIPTION
**Pull request by Chronophylos**
`aaa` -> `oaa`
`Angel Arena Allstars` -> `Open Angel Arena`

### CODE REVIEW

**Comment by Chronophylos** [comment]
Do not merge this yet.
I'm waiting on the links.

File: README.md [hunk 1]
```diff
@@ -22,5 +22,5 @@ From within the addon directory, run `gulp`.
 # License
 MIT
 
-[soundcloud-link]: https://soundcloud.com/angelarenaallstars "Music for Angel Arena Allstars"
-[discord-link]: https://discord.gg/ "Angel Arena Allstars Discord Instant Invite"
+[soundcloud-link]: https://soundcloud.com/angelarenaallstars "Music for Open Angel Arena"
+[discord-link]: https://discord.gg/ "Open Angel Arena Discord Instant Invite"
```
Comment (chrisinajar) [inline README.md]: we should also just add the real discord link right into the description. it's already old news so i'm not worried about spam

**Comment by Chronophylos** [comment]
Ok

**Comment by Chronophylos** [comment]
@chrisinajar you can merge this.

**Pull request merged**